controller/src/controller.rs

```diff
@@ -416,64 +443,41 @@ fn node_has_label(node: &Node) -> bool {
         ));
 }
 
+/// Method to verify if brupop scheduler has been set up as time window
+/// Duration between each scheduled time within time window would be 1 sec.
 #[instrument(skip())]
-fn within_time_window() -> Result<bool> {
-    let update_window_start_env = env::var(UPDATE_WINDOW_START_ENV_VAR).context(
-        controllerclient_error::MissingEnvVariableSnafu {
-            variable: UPDATE_WINDOW_START_ENV_VAR.to_string(),
-        },
-    )?;
+fn is_time_window(brupop_schedule: &Schedule) -> Result<bool> {
+    Ok(calculate_duration_between_scheduled_datetimes(brupop_schedule)?.num_seconds() == 1)
```

Comment:
Does this mean `* * * * * * ` is a time window?

Reply:
yes. `* * * * * *`  is a time window